.circleci/config.yml

```diff
@@ -160,6 +140,7 @@ references:
         . build.sh circleci
         mkdir -p workspace
         echo $BUILD_TAG > workspace/build_tag
+        python3 config/pipeline/aws_ecr_scan_results.py --search peoplefinder-ecr --tag $BUILD_TAG --slack_webhook $SLACK_WEBHOOK
```

Comment:
@digitalali-moj , I think the peoplefinder-ecr should be changed to PQ one?

Reply:
you're right! thanks